ethereum-consortium-blockchain-network/nested/fiveBootNodeVMExtension.json

```diff
@@ -0,0 +1,131 @@
+        }
+      }
+    },
+    {
+      "apiVersion": "[parameters('apiVersion')]",
+      "type": "Microsoft.Compute/virtualMachines/extensions",
+      "name": "[concat(parameters('mnVMNamePrefix'), copyIndex(),'/newuserscript')]",
+      "copy": {
+        "name": "bootMNNodesConfigLoop",
+        "count": "[variables('numBootNodes')]"
+      },
+      "location": "[resourceGroup().location]",
+      "properties": {
+        "publisher": "Microsoft.OSTCExtensions",
+        "type": "CustomScriptForLinux",
+        "typeHandlerVersion": "1.4",
+        "autoUpgradeMinorVersion": true,
+        "settings": {
+          "fileUris": [
+            "[concat(parameters('artifactsLocationURL'), '/scripts/configure-geth.sh')]"
+          ],
+          "commandToExecute": "[concat('sh configure-geth.sh ', parameters('adminUsername'), ' ', parameters('ethereumAccountPsswd'), ' ', parameters('privateKeyForPrefundedAccount'), ' ', parameters('artifactsLocationURL'), ' ', parameters('ethereumNetworkID'), ' ', parameters('maxPeers'), ' ', parameters('bootMNNode'), ' ', concat('enode://', parameters('nodeIDArray')[0], '@', reference(concat('Microsoft.Network/networkInterfaces/', parameters('mnNIPrefix'), 0), parameters('apiVersion')).ipConfigurations[0].properties.privateIPAddress, ':', parameters('gethIPCPort'), ',', 'enode://', parameters('nodeIDArray')[1], '@', reference(concat('Microsoft.Network/networkInterfaces/', parameters('mnNIPrefix'), 1), parameters('apiVersion')).ipConfigurations[0].properties.privateIPAddress, ':', parameters('gethIPCPort'), ',', 'enode://', parameters('nodeIDArray')[2], '@', reference(concat('Microsoft.Network/networkInterfaces/', parameters('mnNIPrefix'), 2), parameters('apiVersion')).ipConfigurations[0].properties.privateIPAddress, ':', parameters('gethIPCPort'), ',', 'enode://', parameters('nodeIDArray')[3], '@', reference(concat('Microsoft.Network/networkInterfaces/', parameters('mnNIPrefix'), 3), parameters('apiVersion')).ipConfigurations[0].properties.privateIPAddress, ':', parameters('gethIPCPort'), ',', 'enode://', parameters('nodeIDArray')[4], '@', reference(concat('Microsoft.Network/networkInterfaces/', parameters('mnNIPrefix'), 4), parameters('apiVersion')).ipConfigurations[0].properties.privateIPAddress, ':', parameters('gethIPCPort')), ' ', parameters('numMNNodes'), ' ', parameters('nodeKeyArray')[copyIndex()])]"
```

Comment:
Since this has password and private key, this command to execute should be part of protected settings. Needs to be done across all extension templates in this PR.